File: Baekjoon/18406.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner scan = new Scanner(System.in);
        String n = scan.nextLine();
        int left = 0;
        int right = 0;

        for (int i = 0; i < n.length() / 2; i++) left += (n.charAt(i) - '0');
        for (int i = n.length() / 2; i < n.length(); i++) right += (n.charAt(i) - '0');

        if (left == right) System.out.println("LUCKY");
        else System.out.println("READY");
    }
}

File: Baekjoon/18406.py
n = input()

left = 0
right = 0

for i in range(len(n) // 2):
    left += int(n[i])

for i in range(len(n) // 2, len(n)):
    right += int(n[i])

if left == right:
    print("LUCKY")
else:
    print("READY")
